### DISCUSSION TITLE
Wechselrichter (HM-600) wird nicht mehr erkannt

### DISCUSSION THREAD

**Discussion by bjhesk** [tbnobody/OpenDTU#1618] [Q&A] [answered]
Hallo zusammen,

ich kämpfe mit dem o.g. Problem seit ca. Mitte Dezember.

**Problem:**
"Plötzlich" verbindet sich OpenDTU nicht mehr mit meinem Wechselrichter. Am Tag zuvor hat es noch funktioniert. Ich habe kein Update o.Ä. eingespielt. Allerdings meine ich nun, dass es evtl. mit einem Stromausfall zusammenhängt, der hier in der Stadt aufgetreten ist.

**Was ich bisher gemacht habe:**
Ich habe den ESP32 ganz neu geflasht und alles neu eingetragen.

**Welche Fragen ich mir stelle:**
* Wie kann ich herausfinden, ob durch den Stromausfall die Hardware (ESP oder nRF24L01+) beschädigt wurde?
* Welche Schritte zum Debugging kann ich vornehmen?

Ich würde mich sehr über eure Hilfe freuen. Ich habe schon ganz viel recherchiert, aber ich komme nun leider nicht mehr weiter...

Viele Grüße
B

Hier ein paar Screens:
![grafik](https://github.com/tbnobody/OpenDTU/assets/134172050/ac8f07b5-1fd8-4c41-9a88-de6403e4c195)
![grafik](https://github.com/tbnobody/OpenDTU/assets/134172050/b506b39d-a8fc-4e39-9395-388ed4329c39)

**Comment by teletapi** [comment]
![294218918-b506b39d-a8fc-4e39-9395-388ed4329c39](https://github.com/tbnobody/OpenDTU/assets/94724873/f20d8a28-957f-42dd-bde2-d86a9a68b173)

Warum sind die beiden doppelt belegt..?

**Reply by bjhesk** [reply]
Ich habe das Profil nodemcu_esp32.json ausgewählt: https://github.com/tbnobody/OpenDTU/tree/master/docs/DeviceProfiles

`{
        "name": "NRF24",
        "nrf24": {
            "miso": 19,
            "mosi": 23,
            "clk": 18,
            "irq": 16,
            "en": 4,
            "cs": 5
        },
        "eth": {
            "enabled": false
        }
    },`

**Reply by tbnobody** [reply]
Dadurch dass `"eth": { "enabled": false }` spielen die doppelt vergebenen Pins am ETH interface keine Rolle.

Was siehst du denn z.B. unter "Info" --> "Console"
(Wobei es jetzt vmtl. dunkel ist und der Inverter aus ist)

**Reply by bjhesk** [reply]
Danke schon einmal für eure Rückmeldungen! Hilft schon mal, wenn man weiß, dass man nicht alleine mit seinem Problem ist!

Hier die Ausgabe aus der Konsole:
RX Period End
16:29:46.102 > All missing
16:29:46.102 > Nothing received, resend whole request
16:29:46.102 > TX RealTimeRunData Channel: 3 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 96 CE E7 00 00 00 00 00 00 00 00 F1 45 D2 
16:29:46.603 > RX Period End
16:29:46.603 > All missing
16:29:46.603 > Nothing received, resend count exeeded
16:29:46.706 > TX AlarmData Channel: 23 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 96 CE E7 00 00 00 00 00 00 00 00 2B 5E 09 
16:29:46.760 > Websocket: [/livedata][3] disconnect
16:29:47.567 > RX Period End
16:29:47.567 > All missing
16:29:47.567 > Nothing received, resend whole request
16:29:47.567 > TX AlarmData Channel: 40 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 96 CE E7 00 00 00 00 00 00 00 00 2B 5E 09 
16:29:48.434 > RX Period End
16:29:48.434 > All missing
16:29:48.434 > Nothing received, resend whole request
16:29:48.434 > TX AlarmData Channel: 61 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 96 CE E7 00 00 00 00 00 00 00 00 2B 5E 09 
16:29:49.286 > RX Period End
16:29:49.286 > All missing
16:29:49.286 > Nothing received, resend whole request
16:29:49.286 > TX AlarmData Channel: 75 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 96 CE E7 00 00 00 00 00 00 00 00 2B 5E 09 
16:29:50.145 > RX Period End
16:29:50.145 > All missing
16:29:50.145 > Nothing received, resend whole request
16:29:50.145 > TX AlarmData Channel: 3 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 96 CE E7 00 00 00 00 00 00 00 00 2B 5E 09 
16:29:50.897 > RX Period End
16:29:50.897 > All missing
16:29:50.897 > Nothing received, resend count exeeded
16:29:50.897 > Fetch inverter: 114183******
16:29:51.039 > TX RealTimeRunData Channel: 23 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 96 CE EF 00 00 00 00 00 00 00 00 31 22 7D 
16:29:51.612 > RX Period End
16:29:51.612 > All missing
16:29:51.612 > Nothing received, resend whole request
16:29:51.612 > TX RealTimeRunData Channel: 40 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 96 CE EF 00 00 00 00 00 00 00 00 31 22 7D 
16:29:52.222 > RX Period End
16:29:52.222 > All missing
16:29:52.222 > Nothing received, resend whole request
16:29:52.222 > TX RealTimeRunData Channel: 61 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 96 CE EF 00 00 00 00 00 00 00 00 31 22 7D 
16:29:52.832 > RX Period End
16:29:52.832 > All missing
16:29:52.832 > Nothing received, resend whole request
16:29:52.832 > TX RealTimeRunData Channel: 75 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 96 CE EF 00 00 00 00 00 00 00 00 31 22 7D 
16:29:53.443 > RX Period End
16:29:53.443 > All missing
16:29:53.443 > Nothing received, resend whole request
16:29:53.443 > TX RealTimeRunData Channel: 3 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 96 CE EF 00 00 00 00 00 00 00 00 31 22 7D 
16:29:53.942 > RX Period End
16:29:53.942 > All missing
16:29:53.942 > Nothing received, resend count exeeded
16:29:54.046 > TX AlarmData Channel: 23 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 96 CE EF 00 00 00 00 00 00 00 00 EB 39 A6 
16:29:54.907 > RX Period End
16:29:54.907 > All missing
16:29:54.907 > Nothing received, resend whole request
16:29:54.907 > TX AlarmData Channel: 40 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 96 CE EF 00 00 00 00 00 00 00 00 EB 39 A6 
16:29:55.768 > RX Period End
16:29:55.768 > All missing
16:29:55.768 > Nothing received, resend whole request
16:29:55.768 > TX AlarmData Channel: 61 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 96 CE EF 00 00 00 00 00 00 00 00 EB 39 A6 
16:29:56.629 > RX Period End
16:29:56.629 > All missing
16:29:56.629 > Nothing received, resend whole request
16:29:56.629 > TX AlarmData Channel: 75 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 96 CE EF 00 00 00 00 00 00 00 00 EB 39 A6 
16:29:57.489 > RX Period End
16:29:57.489 > All missing
16:29:57.489 > Nothing received, resend whole request
16:29:57.489 > TX AlarmData Channel: 3 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 96 CE EF 00 00 00 00 00 00 00 00 EB 39 A6 
16:29:58.242 > RX Period End
16:29:58.242 > All missing
16:29:58.242 > Nothing received, resend count exeeded
16:29:58.242 > Fetch inverter: 11418****
16:29:58.344 > TX RealTimeRunData Channel: 23 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 96 CE F6 00 00 00 00 00 00 00 00 A1 85 53 
16:29:58.954 > RX Period End
16:29:58.954 > All missing
16:29:58.954 > Nothing received, resend whole request
16:29:58.954 > TX RealTimeRunData Channel: 40 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 96 CE F6 00 00 00 00 00 00 00 00 A1 85 53 
16:29:59.569 > RX Period End
16:29:59.569 > All missing
16:29:59.569 > Nothing received, resend whole request
16:29:59.569 > TX RealTimeRunData Channel: 61 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 96 CE F6 00 00 00 00 00 00 00 00 A1 85 53 
16:30:00.174 > RX Period End
16:30:00.174 > All missing
16:30:00.174 > Nothing received, resend whole request
16:30:00.174 > TX RealTimeRunData Channel: 75 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 96 CE F6 00 00 00 00 00 00 00 00 A1 85 53 
16:30:00.784 > RX Period End
16:30:00.784 > All missing
16:30:00.784 > Nothing received, resend whole request
16:30:00.784 > TX RealTimeRunData Channel: 3 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 96 CE F6 00 00 00 00 00 00 00 00 A1 85 53 
16:30:01.287 > RX Period End
16:30:01.287 > All missing
16:30:01.287 > Nothing received, resend count exeeded
16:30:01.391 > TX AlarmData Channel: 23 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 96 CE F6 00 00 00 00 00 00 00 00 7B 9E 88 
16:30:02.252 > RX Period End
16:30:02.252 > All missing
16:30:02.252 > Nothing received, resend whole request
16:30:02.252 > TX AlarmData Channel: 40 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 96 CE F6 00 00 00 00 00 00 00 00 7B 9E 88 
16:30:03.119 > RX Period End
16:30:03.119 > All missing
16:30:03.119 > Nothing received, resend whole request
16:30:03.119 > TX AlarmData Channel: 61 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 96 CE F6 00 00 00 00 00 00 00 00 7B 9E 88 
16:30:03.976 > RX Period End
16:30:03.976 > All missing
16:30:03.976 > Nothing received, resend whole request
16:30:03.976 > TX AlarmData Channel: 75 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 96 CE F6 00 00 00 00 00 00 00 00 7B 9E 88 
16:30:04.831 > RX Period End
16:30:04.831 > All missing
16:30:04.831 > Nothing received, resend whole request
16:30:04.831 > TX AlarmData Channel: 3 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 96 CE F6 00 00 00 00 00 00 00 00 7B 9E 88 
16:30:05.588 > RX Period End
16:30:05.588 > All missing
16:30:05.588 > Nothing received, resend count exeeded
16:30:05.588 > Fetch inverter: 11418****
16:30:05.689 > TX RealTimeRunData Channel: 23 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 96 CE FE 00 00 00 00 00 00 00 00 61 E2 FC 
16:30:06.302 > RX Period End
16:30:06.302 > All missing
16:30:06.302 > Nothing received, resend whole request
16:30:06.302 > TX RealTimeRunData Channel: 40 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 96 CE FE 00 00 00 00 00 00 00 00 61 E2 FC 
16:30:06.909 > RX Period End
16:30:06.909 > All missing
16:30:06.909 > Nothing received, resend whole request
16:30:06.909 > TX RealTimeRunData Channel: 61 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 96 CE FE 00 00 00 00 00 00 00 00 61 E2 FC 
16:30:07.522 > RX Period End
16:30:07.522 > All missing
16:30:07.522 > Nothing received, resend whole request
16:30:07.522 > TX RealTimeRunData Channel: 75 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 96 CE FE 00 00 00 00 00 00 00 00 61 E2 FC 
16:30:08.129 > RX Period End
16:30:08.129 > All missing
16:30:08.129 > Nothing received, resend whole request
16:30:08.129 > TX RealTimeRunData Channel: 3 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 96 CE FE 00 00 00 00 00 00 00 00 61 E2 FC 
16:30:08.634 > RX Period End
16:30:08.634 > All missing
16:30:08.634 > Nothing received, resend count exeeded
16:30:08.733 > TX AlarmData Channel: 23 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 96 CE FE 00 00 00 00 00 00 00 00 BB F9 27 
16:30:09.596 > RX Period End
16:30:09.596 > All missing
16:30:09.596 > Nothing received, resend whole request
16:30:09.596 > TX AlarmData Channel: 40 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 96 CE FE 00 00 00 00 00 00 00 00 BB F9 27 
16:30:10.454 > RX Period End
16:30:10.454 > All missing
16:30:10.454 > Nothing received, resend whole request
16:30:10.454 > TX AlarmData Channel: 61 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 96 CE FE 00 00 00 00 00 00 00 00 BB F9 27

**Reply by tbnobody** [reply]
Leider bringt die Ausgabe bei Dunkelheit nichts da der Inverter in diesem Fall nichts sendet. Das Funkmodul ist nur aktiv wenn die LED am Wechselrichter aktiv ist.

**Reply by bjhesk** [reply]
OK. Ich poste morgen nochmal eine Ausgabe, wenn ich weiß, dass der Wechselrichter an ist. Danke bis hierhin!

**Reply by bjhesk** [reply]
Hier kommt die neue Ausgabe. Der WR produziert gerade.

RX Period End
11:06:27.719 > All missing
11:06:27.719 > Nothing received, resend count exeeded
11:06:27.820 > TX AlarmData Channel: 23 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 97 D4 A0 00 00 00 00 00 00 00 00 91 6C DD 
11:06:28.681 > RX Period End
11:06:28.682 > All missing
11:06:28.682 > Nothing received, resend whole request
11:06:28.682 > TX AlarmData Channel: 40 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 97 D4 A0 00 00 00 00 00 00 00 00 91 6C DD 
11:06:28.828 > Admin AP remaining seconds: 10 / 180
11:06:29.542 > RX Period End
11:06:29.542 > All missing
11:06:29.542 > Nothing received, resend whole request
11:06:29.542 > TX AlarmData Channel: 61 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 97 D4 A0 00 00 00 00 00 00 00 00 91 6C DD 
11:06:30.402 > RX Period End
11:06:30.402 > All missing
11:06:30.402 > Nothing received, resend whole request
11:06:30.402 > TX AlarmData Channel: 75 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 97 D4 A0 00 00 00 00 00 00 00 00 91 6C DD 
11:06:31.266 > RX Period End
11:06:31.266 > All missing
11:06:31.266 > Nothing received, resend whole request
11:06:31.266 > TX AlarmData Channel: 3 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 97 D4 A0 00 00 00 00 00 00 00 00 91 6C DD 
11:06:32.071 > RX Period End
11:06:32.071 > All missing
11:06:32.071 > Nothing received, resend count exeeded
11:06:32.071 > Fetch inverter: 11418***
11:06:32.116 > TX RealTimeRunData Channel: 23 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 97 D4 A7 00 00 00 00 00 00 00 00 7B 51 17 
11:06:32.728 > RX Period End
11:06:32.728 > All missing
11:06:32.728 > Nothing received, resend whole request
11:06:32.728 > TX RealTimeRunData Channel: 40 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 97 D4 A7 00 00 00 00 00 00 00 00 7B 51 17 
11:06:33.337 > RX Period End
11:06:33.337 > All missing
11:06:33.337 > Nothing received, resend whole request
11:06:33.337 > TX RealTimeRunData Channel: 61 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 97 D4 A7 00 00 00 00 00 00 00 00 7B 51 17 
11:06:33.943 > RX Period End
11:06:33.943 > All missing
11:06:33.943 > Nothing received, resend whole request
11:06:33.943 > TX RealTimeRunData Channel: 75 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 97 D4 A7 00 00 00 00 00 00 00 00 7B 51 17 
11:06:34.551 > RX Period End
11:06:34.551 > All missing
11:06:34.551 > Nothing received, resend whole request
11:06:34.551 > TX RealTimeRunData Channel: 3 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 97 D4 A7 00 00 00 00 00 00 00 00 7B 51 17 
11:06:35.055 > RX Period End
11:06:35.055 > All missing
11:06:35.055 > Nothing received, resend count exeeded
11:06:35.159 > TX AlarmData Channel: 23 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 97 D4 A7 00 00 00 00 00 00 00 00 A1 4A CC 
11:06:36.019 > RX Period End
11:06:36.019 > All missing
11:06:36.019 > Nothing received, resend whole request
11:06:36.019 > TX AlarmData Channel: 40 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 97 D4 A7 00 00 00 00 00 00 00 00 A1 4A CC 
11:06:36.878 > RX Period End
11:06:36.878 > All missing
11:06:36.878 > Nothing received, resend whole request
11:06:36.878 > TX AlarmData Channel: 61 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 97 D4 A7 00 00 00 00 00 00 00 00 A1 4A CC 
11:06:37.738 > RX Period End
11:06:37.738 > All missing
11:06:37.738 > Nothing received, resend whole request
11:06:37.738 > TX AlarmData Channel: 75 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 97 D4 A7 00 00 00 00 00 00 00 00 A1 4A CC 
11:06:38.599 > RX Period End
11:06:38.599 > All missing
11:06:38.599 > Nothing received, resend whole request
11:06:38.599 > TX AlarmData Channel: 3 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 97 D4 A7 00 00 00 00 00 00 00 00 A1 4A CC 
11:06:38.883 > Admin AP remaining seconds: 20 / 180
11:06:39.362 > RX Period End
11:06:39.362 > All missing
11:06:39.362 > Nothing received, resend count exeeded
11:06:39.362 > Fetch inverter: 11418***
11:06:39.457 > TX RealTimeRunData Channel: 23 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 97 D4 AF 00 00 00 00 00 00 00 00 BB 36 B8 
11:06:40.066 > RX Period End
11:06:40.066 > All missing
11:06:40.066 > Nothing received, resend whole request
11:06:40.066 > TX RealTimeRunData Channel: 40 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 97 D4 AF 00 00 00 00 00 00 00 00 BB 36 B8 
11:06:40.677 > RX Period End
11:06:40.677 > All missing
11:06:40.677 > Nothing received, resend whole request
11:06:40.677 > TX RealTimeRunData Channel: 61 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 97 D4 AF 00 00 00 00 00 00 00 00 BB 36 B8 
11:06:41.285 > RX Period End
11:06:41.285 > All missing
11:06:41.285 > Nothing received, resend whole request
11:06:41.285 > TX RealTimeRunData Channel: 75 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 97 D4 AF 00 00 00 00 00 00 00 00 BB 36 B8 
11:06:41.898 > RX Period End
11:06:41.898 > All missing
11:06:41.898 > Nothing received, resend whole request
11:06:41.898 > TX RealTimeRunData Channel: 3 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 97 D4 AF 00 00 00 00 00 00 00 00 BB 36 B8 
11:06:42.400 > RX Period End
11:06:42.400 > All missing
11:06:42.400 > Nothing received, resend count exeeded
11:06:42.509 > TX AlarmData Channel: 23 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 97 D4 AF 00 00 00 00 00 00 00 00 61 2D 63 
11:06:43.362 > RX Period End
11:06:43.362 > All missing
11:06:43.362 > Nothing received, resend whole request
11:06:43.362 > TX AlarmData Channel: 40 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 97 D4 AF 00 00 00 00 00 00 00 00 61 2D 63 
11:06:44.224 > RX Period End
11:06:44.224 > All missing
11:06:44.224 > Nothing received, resend whole request
11:06:44.224 > TX AlarmData Channel: 61 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 97 D4 AF 00 00 00 00 00 00 00 00 61 2D 63 
11:06:45.082 > RX Period End
11:06:45.082 > All missing
11:06:45.082 > Nothing received, resend whole request
11:06:45.082 > TX AlarmData Channel: 75 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 97 D4 AF 00 00 00 00 00 00 00 00 61 2D 63 
11:06:45.946 > RX Period End
11:06:45.946 > All missing
11:06:45.946 > Nothing received, resend whole request
11:06:45.946 > TX AlarmData Channel: 3 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 97 D4 AF 00 00 00 00 00 00 00 00 61 2D 63 
11:06:46.698 > RX Period End
11:06:46.698 > All missing
11:06:46.698 > Nothing received, resend count exeeded
11:06:46.698 > Fetch inverter: 11418***
11:06:46.804 > TX RealTimeRunData Channel: 23 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 97 D4 B6 00 00 00 00 00 00 00 00 2B 91 96 
11:06:47.485 > RX Period End
11:06:47.485 > All missing
11:06:47.485 > Nothing received, resend whole request
11:06:47.485 > TX RealTimeRunData Channel: 40 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 97 D4 B6 00 00 00 00 00 00 00 00 2B 91 96 
11:06:48.020 > RX Period End
11:06:48.020 > All missing
11:06:48.020 > Nothing received, resend whole request
11:06:48.020 > TX RealTimeRunData Channel: 61 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 97 D4 B6 00 00 00 00 00 00 00 00 2B 91 96 
11:06:48.630 > RX Period End
11:06:48.630 > All missing
11:06:48.630 > Nothing received, resend whole request
11:06:48.630 > TX RealTimeRunData Channel: 75 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 97 D4 B6 00 00 00 00 00 00 00 00 2B 91 96 
11:06:49.024 > Admin AP remaining seconds: 30 / 180
11:06:49.239 > RX Period End
11:06:49.239 > All missing
11:06:49.239 > Nothing received, resend whole request
11:06:49.239 > TX RealTimeRunData Channel: 3 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 97 D4 B6 00 00 00 00 00 00 00 00 2B 91 96 
11:06:49.744 > RX Period End
11:06:49.744 > All missing
11:06:49.744 > Nothing received, resend count exeeded
11:06:49.847 > TX AlarmData Channel: 23 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 97 D4 B6 00 00 00 00 00 00 00 00 F1 8A 4D 
11:06:50.707 > RX Period End
11:06:50.707 > All missing
11:06:50.707 > Nothing received, resend whole request
11:06:50.707 > TX AlarmData Channel: 40 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 97 D4 B6 00 00 00 00 00 00 00 00 F1 8A 4D 
11:06:51.569 > RX Period End
11:06:51.569 > All missing
11:06:51.569 > Nothing received, resend whole request
11:06:51.569 > TX AlarmData Channel: 61 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 97 D4 B6 00 00 00 00 00 00 00 00 F1 8A 4D 
11:06:52.426 > RX Period End
11:06:52.426 > All missing
11:06:52.426 > Nothing received, resend whole request
11:06:52.426 > TX AlarmData Channel: 75 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 97 D4 B6 00 00 00 00 00 00 00 00 F1 8A 4D 
11:06:53.288 > RX Period End
11:06:53.288 > All missing
11:06:53.288 > Nothing received, resend whole request
11:06:53.288 > TX AlarmData Channel: 3 --> 15 83 71 85 93 80 14 72 20 80 11 00 65 97 D4 B6 00 00 00 00 00 00 00 00 F1 8A 4D 
11:06:54.039 > RX Period End
11:06:54.040 > All missing
11:06:54.040 > Nothing received, resend count exeeded
11:06:54.040 > Fetch inverter: 11418***
11:06:54.146 > TX RealTimeRunData Channel: 23 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 97 D4 BD 00 00 00 00 00 00 00 00 1B E2 DE 
11:06:54.759 > RX Period End
11:06:54.759 > All missing
11:06:54.759 > Nothing received, resend whole request
11:06:54.759 > TX RealTimeRunData Channel: 40 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 97 D4 BD 00 00 00 00 00 00 00 00 1B E2 DE 
11:06:55.365 > RX Period End
11:06:55.365 > All missing
11:06:55.365 > Nothing received, resend whole request
11:06:55.365 > TX RealTimeRunData Channel: 61 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 97 D4 BD 00 00 00 00 00 00 00 00 1B E2 DE 
11:06:55.974 > RX Period End
11:06:55.974 > All missing
11:06:55.974 > Nothing received, resend whole request
11:06:55.974 > TX RealTimeRunData Channel: 75 --> 15 83 71 85 93 80 14 72 20 80 0B 00 65 97 D4 BD 00 00 00 00 00 00 00 00 1B E2 DE

**Comment by teletapi** [comment]
![Screenshot_2](https://github.com/tbnobody/OpenDTU/assets/94724873/61e4c9b5-ed35-4856-a404-f72ea706c2d1)
Welche Firmware version hast du geflasht...?
Es gibt da wohl ab und an Probleme die erste oder 2te ESP version zu nehmen immer wieder lese ich das eine der beiden nicht funktioniert.
Die Gründe dafür kenne ich aber nicht. Aber probieren geht über studieren
und um ehrlich zu sein kenne ich den Unterschied auch nicht zwischen generic_esp32 und generic, zumal Opendtu ja den ESP32 sowieso vorraussetzt

**Reply by bjhesk** [reply]
Habe mich für die esp32 entschieden.

![grafik](https://github.com/tbnobody/OpenDTU/assets/134172050/14592a30-8f6a-43c5-a0a5-f4760635aac7)

**Reply by tbnobody** [reply]
Die Unterschiede stehen z.B. in der Doku https://tbnobody.github.io/OpenDTU-docs/firmware/flash_esp/#overview-of-pre-compiled-binaries

**Comment by bjhesk** [comment, marked as answer]
Ich glaube ich habe die Lösung gefunden. Ich hatte noch ein Ersatz-Bauteil für den nRF24L01+! Montiert und läuft!

Wahrscheinlich ist das alte Bauteil defekt. Kann man das irgendwie sicher feststellen?

**Reply by stefan123t** [reply]
@bjhesk gratuliere dass es jetzt wieder geht. Entweder die Ursache war eine Hardware (Verkabelung) oder Software (Konfiguration) Frage.

Wenn jetzt alles läuft, könntest Du ggf. nochmal das alte NRF24 Modul anschliessen und so prüfen ob es wirklich kaputt ist oder nicht.

**Reply by bjhesk** [reply]
Ich habe die Kabel 1:1 umgesteckt auf das neue Modul und auch an der Konfiguration nichts geändert. Das alte Modul ist schon im Müll... Wäre interessant gewesen, ob man es dem kaputten Modul ansehen kann, dass es kaputt ist. Beim nächsten Mal hab ich jetzt nämlich kein Ersatzmodul mehr. :-)
